test/test_cuda.py

```diff
@@ -5169,6 +5181,40 @@ def test_graph_grad_scaling(self, device, dtype, optim_info, foreach, fused):
             self.assertEqual(scaler._growth_tracker, growth_tracker)
 
 
+class TestGDS(TestCase):
+    def _get_tmp_dir_fs_type(self):
+        my_path = os.path.realpath("/tmp")
+        root_type = ""
+        for part in psutil.disk_partitions():
+            if part.mountpoint == "/":
+                root_type = part.fstype
+                continue
+            if part.mountpoint == my_path:
```

Comment:
Should this be startswith ?

Reply:
the path is "/tmp", so I think part.mountpoint == my_path is fine (if `/tmp` is not a mountpoint), then it will return the root_type at the end